app/frontend/components/DonationBands/DonationBands.css

```diff
@@ -0,0 +1,34 @@
+.DonationBands-container {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  text-align: left;
+  margin-bottom: 12px;
+}
+
+.AmountSelection__proceed-button, .AmountSelection__currency-selector {
+  margin-top: 12px;
+}
+.DonationBands-button, .DonationBands-input {
+  width: 48.5%;
+  display: inline-block;
+  font-size: 24px;
+}
+
+.DonationBands__input {
+  height: 60px;
+  margin: 0;
+  padding: 0;
+  border-radius: 3px;
+  border-width: 0;
+  text-align: center;
+  font-size: 20px;
+  width: 48.5% !important;
```

Comment:
nah mate you just had to change the selector here to `input.DonationBands__input` 🍴